probe-rs/src/debug/debug_info.rs

```diff
@@ -2084,4 +2129,60 @@ mod test {
 
         insta::assert_snapshot!(printed_backtrace);
     }
+    #[test]
+    fn probe_rs_debug_unwind_tests() {
+        // TODO: Add RISC-V tests.
+        for chip_name in ["nRF52833_xxAA", "RP2040"] {
```

Comment:
We should use https://crates.io/crates/test-case for this :)

Reply:
I looked at that initially, but couldn't see the value of adding an additional dependency just so I can drive a `for loop` that will never have much more than a hand full of array members.

Are there other examples in our tests that would justify the dependency? Should we maybe get together and rationalize our tests?

Reply:
So I think we should have one test per one test case/variation. Your current test runs two test cases in one test which only generates one entry, does not run in parallel and is harder to debug where the issue is.
I think the additional dep is warranted and I am looking forward to use it a lot in the future :)